src/dialogs/menu.c

```diff
@@ -855,20 +856,37 @@ pass_uri_to_command(struct session *ses, struct document_view *doc_view,
 		return FRAME_EVENT_OK;
 	}
 
+	printf("              \n                \n");
```

Comment:
This line is debug, right?
Remove it please and I'll merge.

Reply:
Yes, it was debug. It must have come out in git-show, but I didn't spot it.

I also took the opportunity to fix the help: the external command does not receive standard input when running in background, but may still receive input from XWindow, if running under XWindow.